src/main/scala/com/databricks/spark/avro/AvroOutputWriter.scala

```diff
@@ -104,7 +104,13 @@ private[avro] class AvroOutputWriter(
       case DateType => (item: Any) =>
         if (item == null) null else item.asInstanceOf[Date].getTime
       case ArrayType(elementType, _) =>
-        val elementConverter = createConverterToAvro(elementType, structName, recordNamespace)
+        val elementConverter = createConverterToAvro(
+          elementType,
+          structName,
+          elementType match {
```

Comment:
On second thought, I think we can create a function for these four lines of code. 
A function accept elementType, recordNamespace, structName and returns the record name space.
So that we won't have duplicated code.
I am making release today, thanks for the work in advance!

Reply:
This match is also on SchemaConverters, should we create a general function for both classes then? In wich class do you think we can put it?

Reply:
A general function In `SchemaConverters`, I think.